packages/php/email-editor/src/Engine/Renderer/class-renderer.php

```diff
@@ -12,7 +12,7 @@
 use Automattic\WooCommerce\EmailEditor\Engine\Templates\Templates;
 use Automattic\WooCommerce\EmailEditor\Engine\Theme_Controller;
 use Automattic\WooCommerce\EmailEditor\Engine\PersonalizationTags\Personalization_Tags_Registry;
-use Soundasleep\Html2Text;
+use Automattic\WooCommerce\EmailEditor\Engine\Renderer\Html2Text;
```

Comment:
Since it's in the same namespace (`Automattic\WooCommerce\EmailEditor\Engine\Renderer`), I don't think we need the import here.


```suggestion
```